code/modules/organs/organ.dm

```diff
@@ -305,23 +305,17 @@
 
 /obj/item/organ/proc/robotize() //Being used to make robutt hearts, etc
 	robotic = 2
-	src.status &= ~ORGAN_BROKEN
```

Comment:
Why is this stuff being removed?

Reply:
Setting the flag to = ORGAN_ROBOT should already do everything that does, no?